ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -977,6 +987,19 @@ interface Ethernet66
    vrrp 3 ipv4 100.64.0.1
    vrrp 3 ipv4 version 3
 !
+interface Ethernet67
+   description Custom_Transceiver_Frequency
+   no shutdown
+   switchport
+   transceiver frequency 190050.000
+!
+interface Ethernet68
+   description Custom_Transceiver_Frequency
+   no shutdown
+   switchport
+   transceiver frequency 190080.000 ghz
+   transceiver media override 100gbase-ar4
```

Comment:
configs are not in sequence as per EOS cli. override config will come before to frequency
```
wan1-dc1(config-if-Et1)#show active 
interface Ethernet1
   description Custom_Transceiver_Frequency
   mtu 9214
   ip address 10.100.0.129/31
   transceiver media override 100gbase-ar4
   transceiver frequency 190080.000 ghz
```

Reply:
fixed, thanks!